lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -77,24 +83,30 @@ def main():
     # declare the pedestal calibrator
     lst_r0 = LSTR0Corrections(pedestal_path=args.pedestal_file, config=config)
 
+    reader = LSTEventSource(input_url=path_list[0], max_events=args.max_events)
     # declare the time corrector
     timeCorr = TimeCorrectionCalculate(calib_file_path=args.output_file,
                                        config=config,
                                        subarray=reader.subarray)
 
     tel_id = timeCorr.tel_id
 
-    for i, event in enumerate(reader):
-        if event.index.event_id % 5000 == 0:
-            print(event.index.event_id)
+    for i, path in enumerate(path_list):
+        log.info(f'File {i+1} out of {len(path_list)}')
+        log.info(f'Processing: {path}')
+        reader = LSTEventSource(input_url=path, max_events=args.max_events)
+        log.info(f'Number of files in this subrun: {reader.multi_file.num_inputs()}')
```

Comment:
It this the number of `streams` in the subrun? Shouldn't this be always 4?

Reply:
Yes, should be always 4

Reply:
then we probably do not want to print it every loop, right? I would even suggest not to print it at all

Reply:
Yes, you are right. I removed.

Reply:
thanks!